### PR TITLE
docs: update download-and-launch.adoc

### PR DESCRIPTION
**Pull request by marcobonita**
Adding a few details: prerequisites and how to update files

### CODE REVIEW

File: modules/applications/pages/download-and-launch.adoc [hunk 1]
```diff
@@ -6,6 +6,11 @@
 For Subscription editions only.
 ====
 
+== Pre-requisites 
+* Docker installed
+* Nginx installed
```
Comment (rbioteau) [inline modules/applications/pages/download-and-launch.adoc:11]: Isn't nginx setup using a Docker image ?

File: modules/applications/pages/download-and-launch.adoc [hunk 2]
```diff
@@ -175,7 +180,7 @@ Run the following command to launch Bonita UI Builder:
 docker compose up -d
 ----
 
-Then, you can access Bonita UI Builder at `http://localhost`.
+Then, you can access Bonita UI Builder at `http://localhost:8090`.
```
Comment (rbioteau) [inline modules/applications/pages/download-and-launch.adoc:179]: In the demo, the reverse proxy was handling the redirection to the proper port. Did it changed ?

**Review by rbioteau — CHANGES_REQUESTED** [review]

File: modules/applications/pages/download-and-launch.adoc (hunk 2 above)
Comment (marcobonita) [reply modules/applications/pages/download-and-launch.adoc:179]: When I tested it, it didn't redirect me properly to the right URL, hence this change

File: modules/applications/pages/download-and-launch.adoc (hunk 1 above)
Comment (marcobonita) [reply modules/applications/pages/download-and-launch.adoc:11]: I'm not sure. I had to `brew install nginx` and manually create a nginx.conf file

**Review by rbioteau — CHANGES_REQUESTED** [review]
In the Download and launch Bonita UI Builder page.
The docker-compose file should use an antora variable to store the bonita-ui-builder image version.

File: modules/applications/pages/download-and-launch.adoc (hunk 2 above)
Comment (marcobonita) [reply modules/applications/pages/download-and-launch.adoc:179]: OK...it just worked now 🤔

**Comment by marcobonita** [comment]
Some of the proposed changes are not relevant + kind of redundant with https://github.com/bonitasoft/bonita-doc/pull/2739

**Pull request closed without merging**